automatic/graphviz/ver32.tmp

```diff
@@ -0,0 +1,21 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+Package can be verified like this:
+
+1. Go to
+
+   x32: https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi

```

Comment:
@RedBaron2 My previous comment about 2.38 should have been here and there are many more places where it's added.

Reply:
@magjac 
This is auto replaced by AU, but I will blank it out when I get back